src/python/WMCore/WorkQueue/Policy/Start/Block.py

```diff
@@ -227,8 +227,13 @@ def getMaskedBlocks(self, task, dbs, datasetPath):
             
             for slumis in grouper(lumis, 1000):
                 for block in blocks:
-                    slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), run_num=run,
-                                                        lumi_list=slumis, detail=True)
+                    # run_num=1 will not be allowed anymore in this API (only for LFNs, not blocks
+                    # nor dataset), starting on Feb/2016
+                    if run==1:
+                        slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), lumi_list=slumis, detail=True)
```

Comment:
I like your patch :-) I'll build pre4 and run a quick test by tomorrow. If things work fine, you can make a PR and close this one.
Thanks a lot!